src/explore-education-statistics-admin/src/pages/edit-release/components/EditReleasePageTemplate.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ReactNode } from 'react';
+import PreviousNextLinks, {
+  PreviousNextLink,
+} from '@admin/components/PreviousNextLinks';
+import NavLink from '@admin/components/NavLink';
+import Page from '../../../components/Page';
+
+interface Props {
+  releaseId: string;
+  children: ReactNode;
+  publicationTitle: string;
+  previousLink?: PreviousNextLink;
+  nextLink?: PreviousNextLink;
+}
+
+const EditReleasePageTemplate = ({
+  releaseId,
+  publicationTitle,
+  children,
+  previousLink,
+  nextLink,
+}: Props) => {
+  return (
+    <Page
+      wide
+      breadcrumbs={[
+        {
+          link: '/admin-dashboard',
+          name: 'Administrator dashboard',
+        },
+        { name: 'Edit release', link: '#' },
+      ]}
+    >
+      <h1 className="govuk-heading-l">
+        {publicationTitle}
+        <span className="govuk-caption-l">Edit release</span>
+      </h1>
+      <nav className="app-navigation govuk-!-margin-bottom-9">
+        <ul className="app-navigation__list govuk-!-margin-bottom-0">
+          <li>
+            <NavLink
+              to={`/edit-release/${releaseId}/setup`}
+              activeClassName="app-navigation--current-page"
+            >
+              Release setup
+            </NavLink>
+            <NavLink
+              to={`/edit-release/${releaseId}/data`}
+              activeClassName="app-navigation--current-page"
+            >
+              Add / edit data
+            </NavLink>
+            <NavLink
+              to={`/edit-release/${releaseId}/build-tables`}
+              activeClassName="app-navigation--current-page"
+            >
+              Build tables
+            </NavLink>
+            <NavLink
+              to={`/edit-release/${releaseId}/tables`}
+              activeClassName="app-navigation--current-page"
+            >
+              View / edit tables
+            </NavLink>
+            <NavLink
+              to={`/edit-release/${releaseId}/content`}
+              activeClassName="app-navigation--current-page"
+            >
+              Add / edit content
+            </NavLink>
+            <NavLink
+              to={`/edit-release/${releaseId}/publish-status`}
+              activeClassName="app-navigation--current-page"
+            >
+              Set publish status
+            </NavLink>
+          </li>
+        </ul>
+      </nav>
+      {children}
+      <PreviousNextLinks
+        previousSection={previousLink}
```

Comment:
As we know the order of the routes already, I think we can very well cut down on having to define previous/next links explicitly as we can obtain the current location and find the matching route (and consequently know what the previous/next route should be).

Reply:
Aye but that's what I basically had before - thought you weren't keen as it felt too much like trying to engineer a solution that had been built and it was too complex?

Reply:
Plus I'd have to make the EditReleasePageTemplate have access to RouteComponentProps so it could infer the location, or otherwise from each section page pass the "current route" through, which again is what I had before.

Reply:
Yes this aspect would be pretty similar, but my point was that we weren't really leveraging the router enough. You can actually use the `withRouter` HOC to pass through the route instead if you're not keen on passing it through as a prop

Reply:
Ahh yeah I've used that bad boy before.  OK fair enough, I will look to use this ta

Reply:
Done